docs/staticAnalysis.md

```diff
@@ -89,7 +89,7 @@ The plugin for IntelliJ can be found [here](https://plugins.jetbrains.com/idea/p
 1. In the `Options` tab, set `Target JDK` to 1.7.
 1. Click `OK`.
 
-#####Suppressing PMD warnings
+##### Suppressing PMD warnings
```

Comment:
@AnaghHegde @xpdavid updated in https://github.com/TEAMMATES/teammates/pull/6853/commits/74c8c4482bf5922bb6e804509fb97bdb8e9eb53e